packages/vue/htmlvalidate/rules/no-navbar-class.spec.ts

```diff
@@ -0,0 +1,21 @@
+import "html-validate/jest";
+import { HtmlValidate } from "html-validate";
+import plugin from "../index";
+
+const htmlvalidate = new HtmlValidate({
+    plugins: [plugin],
+    rules: { "fkui/no-navbar-class": "error" },
+});
+
+it("should report warning when `navbar` class is used", async () => {
```

Comment:
Som tumregel brukar jag försöka lägga in lite negativa tester också, några förslag:

* ska inte varna om klassen inte är satt
* ska inte varna om `class` saknar värde dvs `<div class>` (mest för att det händer när man skriver koden i sin IDE, då vill man inte att regeln kraschar)
* ska inte varna om `class` saknas